docs/tutorial_volume_annotation.md

```diff
@@ -0,0 +1,56 @@
+# Tutorial: Volume Annotation with WEBKNOSSOS
+
+In this tutorial, we will explore the volume annotation features of WEBKNOSSOS. We will cover everything from opening and annotating a published dataset to utilizing both basic and advanced annotation tools. Additionally, I will explain a few things about meshes and provide insights into the layer system of WEBKNOSSOS. Let’s get started!
+
+<!--
+Here is full tutorial video, alternatively continue reading below.
+
+![type:video Tutorial: Volume Annotation with WEBKNOSSOS](TODO add YouTube link) -->
+
+If you already have an account, go to the “featured publications” page. Alternatively, sign up for free in less than a minute on [https://webknossos.org](https://webknossos.org).
+
+Select a dataset from the list of publications to start annotating a demo dataset. You can also open one of your own datasets (if you want to learn how to upload a dataset to WEBKNOSSOS, watch our tutorial on how to upload your own data).
+
+## Creating an Annotation
+
+Click on “create annotation” to begin. Navigate through the dataset and zoom in to find the cell you want to annotate.
+
+![type:video](https://static.webknossos.org/assets/docs/tutorial-volume-annotation/01_create_annotation.mp4){: autoplay loop muted}
+
+Select the brush tool to create your first volume annotation. Brush over a segment. Use the eraser tool to correct mistakes.
+
+![type:video](https://static.webknossos.org/assets/docs/tutorial-volume-annotation/02_brushing_and_erasing.mp4){: autoplay loop muted}
+
+Create a new segment by selecting a new segment ID. Use the trace tool to draw more precise contours.
+
+![type:video](https://static.webknossos.org/assets/docs/tutorial-volume-annotation/03_new_segments_lasso.mp4){: autoplay loop muted}
+
+## Smart Annotation Tools
```

Comment:
I think this section was changed quite a bit by Albane in the original Medium post. The changes are not reflected here yet - not sure whether you want to do that

Reply:
Thanks for the heads up. She just released these changes today. I uploaded the new videos as well